FEXHeaderUtils/FEXHeaderUtils/StringArgumentParser.h

```diff
@@ -8,32 +8,39 @@
 namespace FHU {
 
 /**
- * @brief Parses a string of arguments, returning a vector of string_views.
+ * @brief Parses a shebang string, returning a vector of string_views.
```

Comment:
Sadly this heavy change in semantics makes ParseArgumentsFromString a misnomer. It's no longer a generic argument parser, so the function/header file/unit test should be renamed.